frontend/src/components/VSafeBrowsing/VSafeBrowsing.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+  <section id="safe-browsing" aria-labelledby="safe-browsing-heading">
+    <header class="relative mb-6 flex items-center justify-between">
+      <h4 id="safe-browsing-headers" class="caption-bold uppercase">
+        {{ $t("filters.safeBrowsing.title") }}
+      </h4>
+    </header>
+    <i18n tag="p" path="filters.safeBrowsing.desc" class="label-regular">
+      <template #sensitive>
+        <VLink :href="sensitivityPath">{{
+          $t("filters.safeBrowsing.sensitive")
+        }}</VLink>
+      </template>
+    </i18n>
+    <form class="safe-browsing-form">
+      <fieldset class="mb-10 mt-8 flex flex-col gap-8">
+        <div v-for="toggle in toggles" :key="toggle.name">
+          <VCheckbox
+            :id="toggle.name"
+            class="flex-row-reverse justify-between"
+            :value="toggle.name"
+            :checked="toggle.state.value"
+            :disabled="isDisabled(toggle.name)"
+            is-switch
+            @change="toggle.switchFn"
+          >
+            <span class="label-bold">{{
+              $t(`filters.safeBrowsing.toggles.${toggle.name}.title`)
+            }}</span>
+          </VCheckbox>
+          <p
+            class="label-regular mt-2"
+            :class="{ 'text-dark-charcoal-40': isDisabled(toggle.name) }"
+          >
+            {{ $t(`filters.safeBrowsing.toggles.${toggle.name}.desc`) }}
+          </p>
+        </div>
+      </fieldset>
+    </form>
+  </section>
+</template>
+
+<script lang="ts">
+import { computed, defineComponent } from "vue"
+import { useContext } from "@nuxtjs/composition-api"
+
+import { useFeatureFlagStore } from "~/stores/feature-flag"
+import { useUiStore } from "~/stores/ui"
+import { ON, OFF } from "~/constants/feature-flag"
+
+import VCheckbox from "~/components/VCheckbox/VCheckbox.vue"
+import VLink from "~/components/VLink.vue"
+
+/**
+ * Contains toggles to determine the users preferences towards fetching results
+ * that may contain sensitive content and subsequently blurring them to prevent
+ * accidental exposure or showing them directly.
+ */
+export default defineComponent({
+  name: "VSafeBrowsing",
+  components: { VCheckbox, VLink },
+  setup() {
+    const { app } = useContext()
+
+    const sensitivityPath = computed(() => app.localePath("/about")) // TODO
```

Comment:
Do we have an issue associated with this? Should it be linked?

Reply:
Yes, there is an issue, #2550. I'll update the `// TODO` to link to that issue.

Reply:
Fixed in 63c2167.